### PR TITLE
Simple private token tutorial cleanup and fixing some paths

### PR DESCRIPTION
**Pull request by critesjosh**
Fixing some things as I went through the tutorial.

- Looks like you removed a directory. In the tutorial it suggests to create a `./contracts` dir and put the private_token_contract in that. the `./contracts` dir was missing.
- Removed a bunch of autogenerated files in ./contracts/private_token_contract/target
- Cleaned up formatting in the contract file

Apologies for massive change to the README, my code formatter removed a bunch of extra spaces.

EDIT:

I am also seeing an error when trying to deploy the contract with `yarn start`. It works if I run `bun ./src/index.ts`.

```
node:internal/deps/undici/undici:11576
    Error.captureStackTrace(err, this);
          ^

TypeError: fetch failed
    at Object.fetch (node:internal/deps/undici/undici:11576:11)
    at async defaultFetch (file:///home/josh/Documents/test/aztecnr-private-token-example/private-token/node_modules/@aztec/foundation/dest/json-rpc/client/json_rpc_client.js:28:16)
    at async request (file:///home/josh/Documents/test/aztecnr-private-token-example/private-token/node_modules/@aztec/foundation/dest/json-rpc/client/json_rpc_client.js:81:21)
    at async DeployMethod.create (file:///home/josh/Documents/test/aztecnr-private-token-example/private-token/node_modules/@aztec/aztec.js/dest/contract_deployer/deploy_method.js:56:9)
    at async DeployMethod.simulate (file:///home/josh/Documents/test/aztecnr-private-token-example/private-token/node_modules/@aztec/aztec.js/dest/contract/base_contract_interaction.js:16:46)
    at async file:///home/josh/Documents/test/aztecnr-private-token-example/private-token/node_modules/@aztec/aztec.js/dest/contract/base_contract_interaction.js:31:36
    at async SentTx.getTxHash (file:///home/josh/Documents/test/aztecnr-private-token-example/private-token/node_modules/@aztec/aztec.js/dest/contract/sent_tx.js:25:16)
    at async DeploySentTx.getTxHash (file:///home/josh/Documents/test/aztecnr-private-token-example/private-token/node_modules/@aztec/aztec.js/dest/contract/sent_tx.js:25:16)
    at async deployContract (file:///home/josh/Documents/test/aztecnr-private-token-example/private-token/dest/src/index.js:12:38) {
  cause: SocketError: other side closed
      at Socket.onSocketEnd (node:internal/deps/undici/undici:9790:26)
      at Socket.emit (node:events:526:35)
      at endReadableNT (node:internal/streams/readable:1359:12)
      at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
    code: 'UND_ERR_SOCKET',
    socket: {
      localAddress: '127.0.0.1',
      localPort: 32868,
      remoteAddress: undefined,
      remotePort: undefined,
      remoteFamily: undefined,
      timeout: undefined,
      bytesWritten: 810290,
      bytesRead: 1153
    }
  }
}

Node.js v18.17.1
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### CODE REVIEW

File: tutorials/simple-private-token/contracts/private_token_contract/Nargo.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[package]
+name = "private_token"
+type = "contract"
+authors = [""]
+compiler_version = "0.11.1"
+
+[dependencies] 
+aztec = { git="https://github.com/AztecProtocol/aztec-packages/", tag="master", directory="yarn-project/noir-libs/aztec-noir" }
+value_note = { git="https://github.com/AztecProtocol/aztec-packages/", tag="master", directory="yarn-project/noir-libs/value-note"}
```
Comment (critesjosh) [inline tutorials/simple-private-token/contracts/private_token_contract/Nargo.toml:9]: The `aztec-cli compile` command was giving me problems until i added a trailing slash on the first part of the url. Can you confirm this works for you?

File: tutorials/simple-private-token/README.md [hunk 2]
```diff
@@ -116,17 +118,17 @@ contract PrivateToken {
     #[aztec(private)]
     fn transfer(amount: Field, recipient: Field) {}
 
-    unconstrained fn getBalance(owner: Field) -> Field {
+    unconstrained fn get_balance(owner: Field) -> Field {
```
Comment (critesjosh) [inline tutorials/simple-private-token/README.md:121]: snake case seems to be what people are going with in the monorepo.

File: tutorials/simple-private-token/README.md [hunk 3]
```diff
@@ -325,15 +336,15 @@ npm install -g @aztec/cli
 
 ### Set up a project
 
-Go the root directory we created in [this section](https://www.notion.so/Tutorial-Write-and-interact-with-your-first-Aztec-nr-contract-9b50d2af9d3c4111a810926a6701894a?pvs=21) and create a new `yarn` project. `npm` works too.
+Go the root directory we created in [this section](#create-a-project) and create a new `yarn` project. `npm` works too.
```
Comment (critesjosh) [inline tutorials/simple-private-token/README.md:339]: Is this link correct?

File: tutorials/simple-private-token/README.md [hunk 4]
```diff
@@ -395,10 +406,10 @@ The Aztec CLI has a compiler that allows you to autogenerate type-safe typescrip
 Generate one for our private token smart contract like this (assuming you are in the project root directory)
 
 ```bash
-aztec-cli compile --typescript ../../src ./contracts/private_token_contract
+aztec-cli compile --typescript ./src ./contracts/private_token_contract
```
Comment (critesjosh) [inline tutorials/simple-private-token/README.md:409]: I had to use this path when in the project root.

File: tutorials/simple-private-token/contracts/private_token_contract/Nargo.toml (hunk 1 above)
Comment (signorecello) [reply tutorials/simple-private-token/contracts/private_token_contract/Nargo.toml:9]: yeah this was due to a change on the aztec-packages, I have a PR open to fix this on @catmcgee's repo

Comment (critesjosh) [reply tutorials/simple-private-token/contracts/private_token_contract/Nargo.toml:9]: Should we keep this one in the dev rel repo as the source of truth?

Comment (signorecello) [reply tutorials/simple-private-token/contracts/private_token_contract/Nargo.toml:9]: yes, I'll drop mine 👍 just a quick fix on this one, directory also changed to `yarn-project/aztec-nr/aztec`

**Pull request merged**